src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarArtists.tsx

```diff
@@ -34,6 +35,13 @@ export class ArtworkSidebarArtists extends React.Component<ArtistsProps> {
         <FollowArtistButton
           artist={artist}
           user={user}
+          trackingData={{
+            modelName: Schema.OwnerType.Artist,
+            context_module: Schema.ContextModule.Sidebar,
+            context_page: "Artwork page",
+            entity_id: artist._id,
+            entity_slug: artist.id,
+          }}
```

Comment:
In this instance this is how we do tracking with Follow, so no need for decorator here 👍